lib/samples/find_route/find_route_sample.dart

```diff
@@ -31,22 +31,14 @@ class _FindRouteSampleState extends State<FindRouteSample> {
   final _stopsGraphicsOverlay = GraphicsOverlay();
   // the graphics overlay for the route.
   final _routeGraphicsOverlay = GraphicsOverlay();
-  // the symbol for the route line.
-  final _routeLineSymbol = SimpleLineSymbol(
-      style: SimpleLineSymbolStyle.solid, color: Colors.blue, width: 5.0);
-  // the symbol for the start and end stops.
-  final _routeStartSymbol = SimpleMarkerSymbol(
-      style: SimpleMarkerSymbolStyle.circle, color: Colors.green, size: 8.0);
-  final _routeEndSymbol = SimpleMarkerSymbol(
-      style: SimpleMarkerSymbolStyle.circle, color: Colors.red, size: 8.0);
   // the stops for the route.
-  final List<Stop> _stops = [];
+  final _stops = <Stop>[];
   // whether the route has been generated.
   var _isRouteGenerated = false;
   // whether the map is ready.
   var _isReady = false;
   // the directions for the route.
-  List<DirectionManeuver> _directions = [];
+  List _directions = <DirectionManeuver>[];
```

Comment:
this should be just `final` in place of `List`

Reply:
I don't think we can declare it to be final since we are reinitializing `_directions ` in setState inside `generateRoute()` and in `resetRoute()`, no?

Reply:
Oh! Right. In that case, I think you can still do `var directions = <DirectionManeuver>[];`.